widgets/blueprintCatalog/src/RepositoryLinkButton.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import Utils from './utils';
+
+interface IconLinkProps {
+    url: string;
+    displayStyle: string;
+}
```

Comment:
As currently we are supporting only `catalog` and `table` `displayStyle`, we should be able to declare it here, to make the props typing more accurate 🎉 

Luckily we can reuse some of thse existing interface from the [./types.ts](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/widgets/blueprintCatalog/src/types.ts) file 🥳 

Interface which is providing us with that exact type is being named as  `BlueprintCatalogWidgetConfiguration`

We could us it in the following way:

```suggestion
import styled from 'styled-components';
import Utils from './utils';
import type {  BlueprintCatalogWidgetConfiguration } from './types';

interface IconLinkProps {
    url: string;
    displayStyle: BlueprintCatalogWidgetConfiguration['displayStyle'];
}
```

or 

```suggestion
import styled from 'styled-components';
import Utils from './utils';
import type {  BlueprintCatalogWidgetConfiguration } from './types';

interface IconLinkProps extends Pick<BlueprintCatalogWidgetConfiguration, 'displayStyle'> {
    url: string;
}
```

Reply:
implemented first version: https://github.com/cloudify-cosmo/cloudify-stage/pull/2239/commits/18cf3e0969139f6ff79cb6fb00fbe1f8001b6d07 :)